modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -436,22 +625,6 @@
           end
         end
 
-        context 'when no mailing address data is found' do
-          it 'responds with bad request' do
-            with_okta_user(scopes) do |auth_header|
-              VCR.use_cassette('evss/claims/claims') do
-                VCR.use_cassette('brd/countries') do
-                  json = JSON.parse(data)
-                  json['data']['attributes']['changeOfAddress'] = {}
-                  data = json.to_json
-                  post submit_path, params: data, headers: auth_header
-                  expect(response).to have_http_status(:unprocessable_entity)
-                end
-              end
-            end
-          end
-        end
-
```

Comment:
Came across this as I started searching for what already was being tested on changeOfAddress. `mailing address` is not a part of the `changeOfAddress` object and since `changeOfAddress` can be empty I was not sure if this was rendered out of date by these updates, or just had an off description. Either way it did not appear to be testing anything.